### DISCUSSION TITLE
How to Indent text of textblock in WPF?

### DISCUSSION THREAD

**Discussion by VahidEra** [dotnet/wpf#7841] [Q&A] [answered]
Hi guys,
"I have a dialogHost in my WPF project with a TextBlock that looks like the image on the right, and I want to have indentation for each paragraph like the image on the left. How can I create it without adding another TextBlock? Unfortunately, the line indent option in TextBlock is disabled."

![2023-05-18_215741](https://github.com/dotnet/wpf/assets/27216710/105942da-9692-4df4-a01d-218767e5b4b8)

Thank you very much.

**Comment by miloush** [comment]
What is the issue with multiple TextBlocks?

**Reply by VahidEra** [reply]
@miloush I want to see if it is possible to do this with a textblock?

**Reply by miloush** [reply, marked as answer]
Technically you can put any content into TextBlock...

```xml
<TextBlock>
    <StackPanel>
       ...
    </StackPanel>
</TextBlock>
```

Does that count?

**Reply by VahidEra** [reply]
@miloush Dera Jan, WOOOOOOOOOW This is interesting. This is what I was looking for.  Thank you for your help, my dear friend
